learn/educators/nodejs.md

```diff
@@ -0,0 +1,127 @@
+---
+Order: 4
+Area: educators
+TOCTitle: Node.js and JavaScript
+ContentId: 0a44fb8f-961c-4524-8fa9-c4c98618a9b5
+PageTitle: Node.js and JavaScript in Visual Studio Code
+DateApproved: 02/17/2021
+MetaDescription: Node.js and JavaScript in Visual Studio Code
+---
+
+# Node.js and JavaScript in Visual Studio Code
+
+Visual Studio Code the most popular free code editor amongst JavaScript developers around the world according to the [State of JS 2020 survey](https://2020.stateofjs.com/en-US/other-tools/#text_editors). It is super customizable and has a rich ecosystem of extensions to help your students learn node.js and JavaScript.
+
+Read below for recommendations for extensions, settings, and a link to a free lesson that you can use in your classes.
+
+## Intro to Web Development at Naresuan University
+
+[Charles Allen](https://th.linkedin.com/in/ajahncharles) is a lecturer at [Naresuan University](https://www.nu.ac.th/) in Thailand, where he teaches several Computer Science and Business courses. He teaches a project-based Web Development course where students build a TypeScript app using the Vue.js framework and Firebase.
+
+### Why Visual Studio Code?
+
+Charles has been using Visual Studio Code to teach for the past 4 years since he inherited the class. In his own words:
```

Comment:
Not sure we need to mention how he became the professor.
Maybe just "Charles has been using Visual Studio Code to teach Web Development for the part four years."